packages/ses/src/permits-intrinsics.js

```diff
@@ -131,8 +131,8 @@ export default function removeUnpermittedIntrinsics(
    * Validate the object's [[prototype]] against a permit.
    */
   function visitPrototype(path, obj, protoName) {
-    if (!isObject(obj)) {
-      throw TypeError(`Object expected: ${path}, ${obj}, ${protoName}`);
+    if (isPrimitive(obj)) {
+      throw TypeError(`Object expected: ${path}, ${String(obj)}, ${protoName}`);
```

Comment:
The better typing of `isPrimitive` (thanks @gibson042) provoked typescript into reporting this genuine but as a type error. So fix this bug is more than a drive-by.